Editor/AGS.Editor/ImportExport.cs

```diff
@@ -277,8 +277,10 @@ public static CompileMessages ImportOldEditorDatFile(string fileName, Game game,
                         new Script("Module" + i + ".asc", moduleScript, name, description, author, moduleVersion, uniqueKey, false));
                 game.RootScriptFolder.Items.Add(scripts);
 
-                int permissions = reader.ReadInt32();
-                int weAreOwner = reader.ReadInt32();
+                ////int permissions = reader.ReadInt32();
+                ////int weAreOwner = reader.ReadInt32();
+                reader.ReadInt32();
+                reader.ReadInt32();

```

Comment:
Could you appened commented name of variable to the actual working code instead like -
<code>
reader.ReadInt32(); // int permissions
</code>

Right now it is not immediately obvious whether commented ReadInt calls correspond to following ones, or supposed to be 2 separate reads.